misc/network.c

```diff
@@ -1399,8 +1404,10 @@ write_stream_connection4 (int fd, void *buf0, int n, int i_opt)
 
     default:
       if (fp->transport || fp->fd != 0)
-        g_message ("Severe bug! Unhandled transport layer %d (fd=%d)",
-                   fp->transport, fd);
+        g_message ("Severe bug! Unhandled transport layer %d (fd=%d). "
+                   "Function %s called from %s.",
+                   fp->transport, fd, nasl_get_plugin_filename (),
+                   nasl_get_function_name ());
```

Comment:
nasl_get_plugin_filename () and nasl_get_function_name () seems to have the wrong order